### PR TITLE
Add email definition - fixes #391

### LINKED ISSUE #391: Email buttons on the careers page don't include the "mailto:" email address

### PR DESCRIPTION
**Pull request by jukecraft**
fixing #391 by defining `site.email` and linking to `office.email` in the career page

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ paginate_path: "/publications/page:num"
 
 sitename: "Codurance"
 subtitle: "craft at heart"
+email: "info@codurance.com"
```
Comment (sandromancuso) [inline _config.yml]: should be hello@codurance.com

Comment (jukecraft) [reply _config.yml]: will fix

**Comment by franziskas** [comment]
@CarlosMChica I will fix it

File: _config.yml (hunk 1 above)
Comment (jukecraft) [reply _config.yml]: fixed

**Comment by jukecraft** [comment]
@CarlosMChica I changed it on the careers site. I also left the email configuration so that the atom feed would link to the `.com` version.

**Comment by CarlosMChica** [comment]
@franziskas ok, cool :)

**Pull request merged**